src/main/java/api/bpartners/annotator/service/event/ExportTaskCreatedService.java

```diff
@@ -0,0 +1,94 @@
+package api.bpartners.annotator.service.event;
+
+import static api.bpartners.annotator.model.exception.ApiException.ExceptionType.SERVER_EXCEPTION;
+import static api.bpartners.annotator.service.utils.TemplateResolverUtils.parseTemplateResolver;
+import static java.util.UUID.randomUUID;
+
+import api.bpartners.annotator.endpoint.event.model.ExportTaskCreated;
+import api.bpartners.annotator.endpoint.rest.model.ExportFormat;
+import api.bpartners.annotator.file.FileWriter;
+import api.bpartners.annotator.mail.Email;
+import api.bpartners.annotator.mail.Mailer;
+import api.bpartners.annotator.model.exception.ApiException;
+import api.bpartners.annotator.repository.model.ExportTask;
+import api.bpartners.annotator.repository.model.Job;
+import api.bpartners.annotator.service.ExportTaskService;
+import api.bpartners.annotator.service.ExportTaskStatusService;
+import api.bpartners.annotator.service.JobExport.ExportService;
+import api.bpartners.annotator.service.JobService;
+import api.bpartners.annotator.service.utils.ByteWriter;
+import jakarta.mail.internet.InternetAddress;
+import java.io.File;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.function.Consumer;
+import lombok.AllArgsConstructor;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.thymeleaf.context.Context;
+
+@Service
+@AllArgsConstructor
+public class ExportTaskCreatedService implements Consumer<ExportTaskCreated> {
+  public static final String JSON_FILE_EXTENSION = ".json";
+  private final Mailer mailer;
```

Comment:
Such a number of dependencies often means that you failed to identify intermediate compenents that group some dependencies. Eg: all those ExportXxxService could be grouped. Similary, ByteWriter and FileWriter could be grouped.